gcp-auth-extension/src/test/java/io/opentelemetry/contrib/gcp/auth/GcpAuthAutoConfigurationCustomizerProviderTest.java

```diff
@@ -680,6 +767,34 @@ private static Stream<Arguments> provideTargetSignalBehaviorTestCases() {
    * indicates that the mocked credentials are configured to provide DUMMY_GCP_QUOTA_PROJECT_ID as
    * the quota project ID.
    */
+  private static Stream<Arguments> provideProjectIdBehaviorTestCases() {
```

Comment:
The javadoc string from line 755 to 769 applies only to `provideQuotaBehaviorTestCases`, please move it so that it sits right above that function. 

`provideProjectIdBehaviorTestCases` would require a *similar*, but *different* doc string.